docs/_documentations/remote-configuring.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: docs
+title: Configuring Codewind remote
+description: Configuring your environment to use Codewind remote
+keywords: users, projects, Kubernetes, LDAP, user management, access management, login, deployment, pod, security, securing Cloud connection
+duration: 5 minutes
+permalink: remoteconfiguring
+type: document
+parent: installoncloud
+order: 1
+---
+
+## Configuring Codewind remote
+
+To securely configure Codewind remote, do the following steps.
+
+1. Optional: Create a namespace to install into by running the following command: `kubectl create namespace <your_namespace>`. If you do not create a new namespace, Codewind uses the current one.
+2. If you are developing on macOS, you must install an ingress load balancer into your Kube environment. For example: 
+   - `kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/mandatory.yaml`
+   - `kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/provider/cloud-generic.yaml`
+   If you are developing on Windows [get map local address from Mark]
+3. Create the remote deployment by entering the following `cwctl` command: 
+   `cwctl install remote \`
+     `--namespace {your_namespace}  \`
+     `--kadminuser admin \`
+     `--kadminpass <keycloakPassword>  \`
+     `--krealm codewind \`
+     `--kclient codewind  \`
+     `--kdevuser developer \`
+     `--kdevpass  <userPassword>`
+
+   For OpenShift, use:
```

Comment:
This line isn't specific to OpenShift, it will apply for any Kubernetes cluster that `Step 2` didn't apply for. I think we need to change the two install lines to:
- Docker Desktop Kubernetes
- Kubernetes

Or something like that.

@markcor11 FYI

Reply:
Discussed with Mark, changed considerably, will require another review.